digdag-cli/src/main/java/io/digdag/cli/client/ShowSession.java

```diff
@@ -110,17 +119,17 @@ private void showSessions(String projName, String workflowName)
 
     private void printSession(RestSession session)
```

Comment:
Yes, we should make the single-item workflow/session/attempt/etc show commands follow the `--format` configuration.